webapp/packages/plugin-connections/src/ConnectionAuthentication/NetworkHandlerAuthForm.tsx

```diff
@@ -70,7 +77,16 @@ export const NetworkHandlerAuthForm = observer<Props>(function NetworkHandlerAut
           id={id + '_savePassword'}
           name="savePassword"
           state={state}
-          label={translate('connections_connection_edit_save_credentials')}
+          label={translate(
+            distributed
+              ? 'connections_connection_authentication_save_credentials_for_user'
+              : 'connections_connection_authentication_save_credentials_for_session',
+          )}
+          title={translate(
+            distributed
+              ? 'connections_connection_authentication_save_credentials_for_user_tooltip'
+              : 'connections_connection_authentication_save_credentials_for_session_tooltip',
+          )}
```

Comment:
not sure that this setting reflects on SSH/SSL

Reply:
it is. all the cases we found and clarified with QA are in the task description